experiment/kind-e2e.sh

```diff
@@ -46,18 +46,27 @@ PATH="${TMP_GOPATH}/bin:${PATH}"
 # build the base image
 # TODO(bentheelder): eliminate this once we publish this image
 kind build base
+
+# possibly enable bazel build caching before building kubernetes
+BAZEL_REMOTE_CACHE_ENABLED=${BAZEL_REMOTE_CACHE_ENABLED:-false}
+if [[ "${BAZEL_REMOTE_CACHE_ENABLED}" == "true" ]]; then
+    # run the script in the kubekins image, do not fail if it fails
+    /usr/local/bin/create_bazel_cache_rcs.sh || true
+fi
+
 # build the node image w/ kubernetes
-kind build node
+kind build node --type=bazel
 
 # make sure we have e2e requirements
-make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
+#make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
+bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
 
 # ginkgo regexes
 FOCUS="${FOCUS:-"\\[Conformance\\]"}"
 SKIP="${SKIP:-"Alpha|Kubectl|\\[(Disruptive|Feature:[^\\]]+|Flaky)\\]"}"
 
 # arguments to kubetest for the e2e
-KUBETEST_ARGS="--provider=skeleton --test --test_args=\"--ginkgo.focus=${FOCUS} --ginkgo.skip=${SKIP}\" --check-version-skew=false"
+KUBETEST_ARGS="--provider=skeleton --test --test_args=\"--ginkgo.focus=${FOCUS} --ginkgo.skip=${SKIP} --report-dir=${WORKSPACE}/_artifacts --disable-log-dump=true\" --check-version-skew=false"
```

Comment:
probably we should come up with a backup for when WORKSPACE is not set for testing it locally. not sure what a good default would be though

Reply:
as is from a kubernetes checkout `WORKSPACE="${PWD}" ./../test-infra/experiment/kind-e2e.sh` works pretty well, and we do have a k/k gitignore for _artifacts, so maybe that. will revisit tomorrow (PST).

Reply:
WORKSPACE now defaults to PWD if not set